src/prop/zero_level_learner.h

```diff
@@ -0,0 +1,86 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Learner for literals asserted at level zero.
```

Comment:
This suggests that this thing is actually responsible for learning anything. It is only taking learnt stuff and printing it...
Also, can you add a bit about how it should be used / works? (either here on in the class comment)

Reply:
I've added doc.